tests/test_client_methods.py

```
@@ -72,6 +72,23 @@ def test_update_ticket(self, mock_zammad_api: Mock) -> None:
 
+    def test_update_ticket_with_customer(self, mock_zammad_api: Mock) -> None:
+        """Test update_ticket method with customer reassignment."""
+        mock_instance = Mock()
+        mock_instance.ticket.update.return_value = {
+            "id": 1,
+            "title": "Updated Title",
+            "customer": {"email": "new@example.com"},
+        }
+        mock_zammad_api.return_value = mock_instance
+
+        client = ZammadClient(url="https://test.zammad.com/api/v1", http_token="test-token")
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "http_token"

(S106)